playbooks/active-mentorship.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Active mentorship
+description: Guide for practicing active mentorship between a mentor and mentee
+---
+
+Every junior team member has a dedicated mentor assigned to them for guidance, questions and general technical
```

Comment:
I would like to suggest that not only junior devs are the ones that need mentorship. 

I think that there should be two categories, as we discussed on our weekly ITF meet:
a) Mentorship for less experienced developers (mentee has a dedicated mentor)
b) Mentorship for more experienced developers - mentee and mentor relation is dynamic: mid (mentor) -> mid, mid (mentor) -> sen

What do you think?

Reply:
I would add to the doc that a junior developer MUST have a dedicated mentor, while any other level MAY request one. Of course the dynamic will be different, more experienced level developers will need less and different mentoring, the topics discussed will be more abstract (or highly specific, technology for example), etc.

I would for now focus on the mentorship program for junior level developers, where the program is most needed and the return value is highest (the cost of leaving junior developers on their own is highest). Do you agree @ibrcko?

Reply:
I agree!

Can you please elaborate on what does "I would focus" mean?

Reply:
You both said valid points and I agree that different mentorship categories exist. 

In this specific document, I would also focus on the "active dedicated mentorship" that we follow in Bornfight. 

Meaning that, @ibrcko , your option A of mentorship is the topic that's covered in this document.

So maybe a better name for this playbook would be "Active mentorship - junior level"